.github/workflows/ci-latest-helm-chart-release.yaml

```diff
@@ -23,17 +23,22 @@ jobs:
       - name: Update Chart.yaml
         run: |
           STABLE_VERSION=`cat STABLE-RELEASE`
-          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/Chart.yaml
-          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/Chart.yaml
+          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/KubeArmor/Chart.yaml
+          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/KubeArmor/Chart.yaml
+
+          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/KubeArmorOperator/Chart.yaml
+          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/KubeArmorOperator/Chart.yaml
 
       - name: Publish Helm chart
         uses: stefanprodan/helm-gh-pages@master
         with:
           # Access token which can push to a different repo in the same org
           token: ${{ secrets.GH_ACCESS_TOKEN }}
-          charts_dir: deployments
+          charts_dir: deployments/helm
           # repo where charts would be published
           owner: kubearmor
           repository: charts
-          branch: gh-pages
+          branch: main
```

Comment:
why main now?

Reply:
Since main is what people see when they open the repo and they don't know that they have to switch the branch.
Though people don't have to see this repo at all anyways but still. :upside_down_face:

Reply:
@daemon1024 Opinion: is it better to make `gh-pages` as the main branch or `main` branch as the github page trigger :thinking:

Reply:
But like we only deploy gh-pages branch right? We can just switch the default branch in repo too?

Reply:
Hmm... Sure. Let's use `gh-pages` and switch default branch in kubearmor/charts. :+1: